<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gamewindow</class>
 <widget class="QDialog" name="Gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CoolFarm</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>991</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="granjeroLabel">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>560</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="plagaLabel">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>440</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>70</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Arbol</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Ubicacion</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Cantidad de frutos</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Monto total</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Perdidos</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Vendidos</string>
    </property>
   </row>
  </widget>
  <widget class="QLabel" name="ovejaLabel">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>449</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cuervoLabel">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>570</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
